nixos/modules/services/networking/openfire.nix

```diff
@@ -2,6 +2,15 @@
 
 with lib;
 
+let
+
+  inherit (pkgs) jre openfire coreutils which gnugrep gawk gnused;
```

Comment:
Actually, only `openfire` is used in the systemd service below. Please just change the occurrence below to `pkgs.openfire` and we'll merge. ;)